CRM/Activity/Form/Activity.php

```diff
@@ -797,25 +797,20 @@ public function buildQuickForm() {
       // form should be frozen for view mode
       $this->freeze();
 
-      $buttons = array();
-      $buttons[] = array(
-        'type' => 'cancel',
-        'name' => ts('Done'),
-      );
-      $this->addButtons($buttons);
+      $this->addButtons([
+        [
+          'type' => 'cancel',
+          'name' => ts('Done'),
+        ]
```

Comment:
@mattwire I think technically there should be a comma here.

Reply:
Ok, I've added a comma